packages/nvidia_nat_memmachine/memmachine_memory_example.ipynb

```
@@ -0,0 +1,937 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"api-keys\"></a>\n",
+    "### 0.3) API Keys\n",
+    "\n",
+    "For this notebook, you will need the following API keys:\n",
+    "\n",
+    "- **NVIDIA Build:** You can obtain an NVIDIA Build API Key by creating an [NVIDIA Build](https://build.nvidia.com) account and generating a key at https://build.nvidia.com/settings/api-keys\n",
```

Ruff (0.14.14)

Redefinition of unused `sys` from line 56: `sys` redefined here

Remove definition: `sys`

(F811)